script.js

```diff
@@ -0,0 +1,254 @@
+        margin: 1.5rem 1.5rem 0.5rem 0;
+    }
+    
+    .show-popup .technologies {
+        display: flex;
+        align-items: center;
+        padding-inline-start: 0;
+        justify-content: space-between;
+        height: 10vw;
+        width: 85vw;
+        font-weight: 300;
+    }
+
+    .show-popup .technologies li {
+        list-style: none;
+        text-align: center;
+        padding-top: 0.7em;
+        padding-bottom: 0.7em;
+        white-space: nowrap;
+        border: solid black 1px;
+        font-size: 3vw;
+        color: #1a2236;
+        width: 25vw;
+    }
+
+    .show-popup .thumbnail {
+        width: 88vw;
+        min-height: 53vw;
+        box-shadow: 0 0 50px 5px black inset;
+    }
+
+    .show-popup .two {
+        background-image: url(../images/project_thumbnail_2.svg);
+    }
+
+    .show-popup .three {
+        background-image: url(../images/project_thumbnail_3.svg);
+    }
+
+    .show-popup .four {
+        background-image: url(../images/project_thumbnail_4.svg);
+    }
+
+    .show-popup .five {
+        background-image: url(../images/project_thumbnail_5.svg);
+    }
+
+    .show-popup .six {
+        background-image: url(../images/project_thumbnail_6.svg);
+    }
+
+    .show-popup p {
+        font-size: 4vw;
+        max-width: 85vw;
+    }
+
+    .show-popup .projectSee {
+        display: flex;
+        flex-direction: column;
+        align-self: center;
+        margin: 0;
+        margin-bottom: 7vw;
+        gap: 10px;
+        padding-right: 12vw;
+    }
+
+    .show-popup button {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin: 0;
+        width: 80vw;
+        font-family: 'Poppins', sans-serif;
+        font-size: 1.2rem;
+        font-weight: 500;
+        background-color: white;
+
+    }
+
+    .show-popup button img {
+        margin: 0.5rem;
+        height: 4vw;
+    }
+    .show-popup button p {
+        margin: 0.5rem;
+    }
+
+    @media only screen and (min-width: 992px) {
+        .show-popup h2 {
+            font-size: 4vw;
+        }
+
+        .show-popup .technologies {
+            width: 40vw;
+            height: 7vw;
+            margin-top: 2vw;
+            margin-bottom: 2vw;
+        }
+
+        .show-popup .technologies li {
+            font-size: 0.9rem;
+            width: 12vw;
+        }
+
+        .show-popup .thumbnail {
+            display: initial;
+        }
+
+        .show-popup .projectSee {
+            flex-direction: row;
+            max-width: 70vw;
+        }
+
+        .show-popup button img {
+        height: 2.5vw;
+    }
+
+        .show-popup p {
+        max-width: 70vw;
+        font-size: 2vw;
+        margin: 5rem 2rem 5rem 8vw;
+        }
+    }
+    `;
+
+function displayProject(num) {
+  document.body.appendChild(projectPopUp);
+  projectPopUp.style.top = '0';
+  projectPopUp.querySelector('div').appendChild(projectList[num].title.cloneNode(true));
```

Comment:
- [ ] Kindly fix this error, the `projectList[num]` at some case equal null so you can't get the title property.

![image](https://user-images.githubusercontent.com/35535130/156371269-0bb51d05-9c28-4b3e-84ae-2047824ee975.png)

Reply:
@keroloussamy. I have tried to fix the dev tools error but all to no avail. when I make a change that doesn't throw the dev tool error, I start having the eslint errors. Do you have any suggestions?

Reply:
@d2ndjim I think the issue with linter can be solve, you can share the error with me

Reply:
@keroloussamy Here is the linters error:  209:10  error ` 'displayProject' is defined but never used  no-unused-vars`

Reply:
@d2ndjim Yes that happened because you didn't call it in the JS file, so linters see that you have defined something useless, you can solve this by instead of `<button class="see-project" onclick="displayProject(2)" type="button">` call the function on the HTML, you can use [addEventListener](https://www.w3schools.com/jsref/met_document_addeventlistener.asp) in the js file and call the method.

Reply:
@keroloussamy I did that and the linters error got solved but now it's an issue in the dev tool. Can I work on this later as there's an upcoming lesson about this in the curriculum on Friday?. I should have studied more on a better way to do this by then. Or if you're willing to jump on a call with me to check it out. I'll great appreciate it.

Reply:
@d2ndjim Try to fix it and if the problem still remains, contact me.

Reply:
@keroloussamy Hi, So I've tried to fix the issues and I still don't have it done yet. It just can't seem to work.